--- httplib2-0.20.4/python2/httplib2/__init__.py.orig	2022-04-28 06:26:39.115490769 +0200
+++ httplib2-0.20.4/python2/httplib2/__init__.py	2022-04-28 06:29:49.124461403 +0200
@@ -962,7 +962,10 @@ def proxy_info_from_url(url, method="htt
     """
     url = urlparse.urlparse(url)
 
-    proxy_type = 3  # socks.PROXY_TYPE_HTTP
+    if method == 'http':
+        proxy_type = 4 # socks.PROXY_TYPE_HTTP_NO_TUNNEL
+    else:
+        proxy_type = 3  # socks.PROXY_TYPE_HTTP
     pi = ProxyInfo(
         proxy_type=proxy_type,
         proxy_host=url.hostname,
