--- httplib2-0.17.0/tests/test_cacerts_from_env.py.orig	2020-01-24 14:57:26.000000000 +0100
+++ httplib2-0.17.0/tests/test_cacerts_from_env.py	2020-03-23 20:44:34.443627324 +0100
@@ -1,10 +1,12 @@
 import os
 import sys
-import mock
 import pytest
 import tempfile
 import httplib2
-
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 CA_CERTS_BUILTIN = os.path.join(os.path.dirname(httplib2.__file__), "cacerts.txt")
 CERTIFI_CERTS_FILE = "unittest_certifi_file"
--- httplib2-0.17.0/tests/test_http.py.orig	2020-01-24 14:57:26.000000000 +0100
+++ httplib2-0.17.0/tests/test_http.py	2020-03-23 20:45:02.520141887 +0100
@@ -5,13 +5,15 @@
 import email.utils
 import errno
 import httplib2
-import mock
 import os
 import pytest
 from six.moves import http_client, urllib
 import socket
 import tests
-
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 def _raise_connection_refused_exception(*args, **kwargs):
     raise socket.error(errno.ECONNREFUSED, "Connection refused.")
--- httplib2-0.17.0/tests/test_other.py.orig	2020-01-24 14:57:26.000000000 +0100
+++ httplib2-0.17.0/tests/test_other.py	2020-03-23 20:45:22.020036247 +0100
@@ -1,5 +1,4 @@
 import httplib2
-import mock
 import os
 import pickle
 import pytest
@@ -8,7 +7,10 @@
 import tests
 import time
 from six.moves import urllib
-
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 @pytest.mark.skipif(
     sys.version_info <= (3,),
--- httplib2-0.17.0/tests/test_proxy.py.orig	2020-01-24 14:57:26.000000000 +0100
+++ httplib2-0.17.0/tests/test_proxy.py	2020-03-23 20:45:42.219926815 +0100
@@ -9,12 +9,15 @@
 from __future__ import print_function
 
 import httplib2
-import mock
 import os
 import pytest
 import socket
 import tests
 from six.moves import urllib
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 
 def _raise_name_not_known_error(*args, **kwargs):
